x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -498,7 +498,44 @@ func kibanaFetchPolicy(cfg setupConfig, client *kibana.Client, streams *cli.IOSt
 	if err != nil {
 		return nil, err
 	}
-	return findPolicy(cfg, policies.Items)
+	packagePolicies, err := kibanaFetchPackagePolicies(cfg, client, streams)
```

Comment:
no, we are grabbing the package policies instead of agent policies. unfortunately the agent policy doesn't have all the information we require to know if a given installed package is a fleet integration.